docs/source/contributing/developing-core.md

```diff
@@ -289,21 +289,59 @@ Used by Volto, you can also use it in other JavaScript frameworks and environmen
 ## Supported frameworks
 
 Plone supports several frontend implementations, the main one being Volto as the default frontend and reference React-based implementation.
-There are plans to support implementations in other frontends, including NextJS and Remix.
+The others are currently under heavy development, marked as experimental and for now, they remain as a proof of concept.
+Although they do work now in an acceptable way, the implementation might change in the future.
+Please note that these implementations only show how to access the public Plone content in the current site and dealing with data fetching and routing.
+They use the Plone Frontend Strategic Packages (`@plone/registry`, `@plone/client`, `@plone/components`, etc).
 
 ### Plone
 
 The default frontend and reference React-based implementation in Plone is Volto.
 In the `apps` folder you'll find a Volto project scaffolding that uses Volto as a library.
```

Comment:
```suggestion
In the `apps` folder, you'll find a Volto project scaffolding that uses Volto as a library.
```

Reply:
@sneridagh can you verify whether this sentence and the next one are still correct, especially since we're kicking Yeoman to the curb.

Reply:
It is correct for now, we will have to change it (or even remove it) when the new setup is a official.